diff --git a/node_modules/react-native-drop-shadow/android/build.gradle b/node_modules/react-native-drop-shadow/android/build.gradle
index 83d9179..9f081ef 100644
--- a/node_modules/react-native-drop-shadow/android/build.gradle
+++ b/node_modules/react-native-drop-shadow/android/build.gradle
@@ -101,6 +101,10 @@ android {
     targetSdkVersion getExtOrIntegerDefault("targetSdkVersion")
     buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", isNewArchitectureEnabled().toString()
   }
+
+  buildFeatures {
+      buildConfig true
+  }
   buildTypes {
     release {
       minifyEnabled false
